lib/internal/source_map/source_map_cache.js

```diff
@@ -91,6 +107,9 @@ function maybeCacheSourceMap(filename, content, cjsModuleInstance, isGeneratedSo
     /\/[*/]#\s+sourceMappingURL=(?<sourceMappingURL>[^\s]+)/,
```

Comment:
Nit, if we're pulling `kSourceURLMagicComment` would be more consistent to do the same for `sourceMappingURL`?